<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLWebcamIntrinsicsWidget</class>
 <widget class="QWidget" name="qMRMLWebcamIntrinsicsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>330</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="ctkCollapsibleButton" name="collapsibleButton_Camera">
     <property name="text">
      <string>Camera</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="qMRMLNodeComboBox" name="comboBox_CameraSelector">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLWebcamNode</string>
         </stringlist>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_CameraMatrix">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Camera Matrix</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidget_CameraMatrix">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
           <property name="gridStyle">
            <enum>Qt::DotLine</enum>
           </property>
           <property name="rowCount">
            <number>3</number>
           </property>
           <property name="columnCount">
            <number>3</number>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <row/>
           <row/>
           <row/>
           <column/>
           <column/>
           <column/>
           <item row="0" column="0">
            <property name="text">
             <string>0.0</string>
            </property>
            <property name="textAlignment">
             <set>AlignTrailing|AlignVCenter</set>
            </property>
           </item>
           <item row="0" column="1">
            <property name="text">
             <string>0.0</string>
            </property>
            <property name="textAlignment">
             <set>AlignTrailing|AlignVCenter</set>
            </property>
           </item>
           <item row="0" column="2">
            <property name="text">
             <string>0.0</string>
            </property>
            <property name="textAlignment">
             <set>AlignTrailing|AlignVCenter</set>
            </property>
           </item>
           <item row="1" column="0">
            <property name="text">
             <string>0.0</string>
            </property>
            <property name="textAlignment">
             <set>AlignTrailing|AlignVCenter</set>
            </property>
           </item>
           <item row="1" column="1">
            <property name="text">
             <string>0.0</string>
            </property>
            <property name="textAlignment">
             <set>AlignTrailing|AlignVCenter</set>
            </property>
           </item>
           <item row="1" column="2">
            <property name="text">
             <string>0.0</string>
            </property>
            <property name="textAlignment">
             <set>AlignTrailing|AlignVCenter</set>
            </property>
           </item>
           <item row="2" column="0">
            <property name="text">
             <string>0.0</string>
            </property>
            <property name="textAlignment">
             <set>AlignTrailing|AlignVCenter</set>
            </property>
           </item>
           <item row="2" column="1">
            <property name="text">
             <string>0.0</string>
            </property>
            <property name="textAlignment">
             <set>AlignTrailing|AlignVCenter</set>
            </property>
           </item>
           <item row="2" column="2">
            <property name="text">
             <string>0.0</string>
            </property>
            <property name="textAlignment">
             <set>AlignTrailing|AlignVCenter</set>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_DistCoeffs">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Distortion Coefficients</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QTableWidget" name="tableWidget_DistCoeffs">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
           <property name="gridStyle">
            <enum>Qt::DotLine</enum>
           </property>
           <property name="rowCount">
            <number>1</number>
           </property>
           <property name="columnCount">
            <number>5</number>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <row/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <item row="0" column="0">
            <property name="text">
             <string>0.0</string>
            </property>
            <property name="textAlignment">
             <set>AlignTrailing|AlignVCenter</set>
            </property>
           </item>
           <item row="0" column="1">
            <property name="text">
             <string>0.0</string>
            </property>
            <property name="textAlignment">
             <set>AlignTrailing|AlignVCenter</set>
            </property>
           </item>
           <item row="0" column="2">
            <property name="text">
             <string>0.0</string>
            </property>
            <property name="textAlignment">
             <set>AlignTrailing|AlignVCenter</set>
            </property>
           </item>
           <item row="0" column="3">
            <property name="text">
             <string>0.0</string>
            </property>
            <property name="textAlignment">
             <set>AlignTrailing|AlignVCenter</set>
            </property>
           </item>
           <item row="0" column="4">
            <property name="text">
             <string>0.0</string>
            </property>
            <property name="textAlignment">
             <set>AlignTrailing|AlignVCenter</set>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
